src/afl-fuzz-init.c

```diff
@@ -2332,7 +2332,7 @@ void setup_dirs_fds(afl_state_t *afl) {
         afl->fsrv.plot_file,
         "# relative_time, cycles_done, cur_item, corpus_count, "
         "pending_total, pending_favs, map_size, saved_crashes, "
-        "saved_hangs, max_depth, execs_per_sec, total_execs, edges_found\n");
+        "saved_hangs, max_depth, execs_per_sec, total_execs, edges_found, total_crashes, servers_count, san1_exec...\n");
```

Comment:
can you please remove the "..." at the end? :-)

Reply:
Done!